docs/installation.rst

```diff
@@ -1,12 +1,12 @@
 Installation
 ============
 
-``openfe`` currently only works on POSIX systems (macOS and UNIX/Linux).
+**openfe** is currently only compatible with POSIX systems (macOS and UNIX/Linux).
 
 We try to follow `SPEC0 <https://scientific-python.org/specs/spec-0000/>`_ as far as minimum supported dependencies, with the following caveats:
 
-- Python 3.10, 3.11, 3.12 - **we do not yet support Python 3.13**
 - OpenMM 8.0, 8.1.1, 8.1.2, 8.2.0 - **we do not yet support OpenMM v8.3.0**
+- `OpenEye Toolkits` is not yet compatible with Python 3.13, so **openfe** cannot use openeye functionality with Python 3.13.
```

Comment:
This is clear! I was going to add a bit about if you want to use the openeye toolkit with openfe, you need to install python 3.12 or something but I think this makes it pretty clear and saves us from having to make some recommendation.

Reply:
I don't mind this being here, but I honestly wouldn't bother. OpenEye functionality is limited to elf10, and we don't advertise it in our docs.